server/views/help.ejs

```diff
@@ -29,55 +29,55 @@
     </tr>
     <tr>
       <td>Delete a deprel</td>
-      <td>Select all the deprels to be removed with right click, then press either <b>del</b> or <b>backspace</b>. If you want to unselect all the arcs you've just selected, press <b>esc</b>.</td>
+      <td>Select all the deprels to be removed with right click, then press either <kbd>del</kbd> or <kbd>backspace</kbd>. If you want to unselect all the arcs you've just selected, press <kbd>esc</kbd>.</td>
```

Comment:
Is `kbd` standard html?  Wouldn't it be better to do something like `<span class="kbd">...</span>`?

Reply:
it's included in Bootstrap framework and works.